app/commands/thredded/autofollow_mentioned_users.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+module Thredded
+  class AutofollowMentionedUsers
+    def initialize(post)
+      @post = post
+    end
+
+    def run
+      members = autofollows
+      return unless members.present?
+    end
+
+    def autofollows
+      user_names = Thredded::AtNotificationExtractor.new(post.content).run
+      members = post.readers_from_user_names(user_names).to_a
+
+      members.delete post.user
+      members = exclude_those_opting_out_of_at_notifications(members)
+
+      members.each do |user|
+        Thredded::UserTopicFollow.create_unique(user.id, @post.postable_id, Thredded::UserTopicFollow::REASON_MENTIONED)
```

Comment:
```ruby
Thredded::UserTopicFollow.create_unique(
  user.id,
  post.postable_id,
  Thredded::UserTopicFollow::REASON_MENTIONED
)
```

That `@post` can turn into `post` since it's an instance attribute per the `attr_reader`